config/policies.js

```diff
@@ -19,24 +19,39 @@
 
 module.exports.policies = {
```

Comment:
These are important, we should double-check/test each function to make sure the permissions are correct.